bin/k8s-read-config

```diff
@@ -33,6 +33,13 @@ else
   exit 1
 fi
 
+if [[ -n ${ROK8S_DEPLOY} ]] && [[ -z ${ROK8S_ENABLE_OPTIONAL_CONFIGS} ]]; then
+    allow_missing_files=false
+else
+    allow_missing_files=true
+fi
+echo missing files ${allow_missing_files} ${ROK8S_DEPLOY} ${ROK8S_ENABLE_OPTIONAL_CONFIGS}
```

Comment:
Nit: we should probably delete this debugging (or alternatively make it clearer so it is less debug-y)

Reply:
Removed!